luminance/src/texture.rs

```diff
@@ -613,6 +613,32 @@ where
     self.size
   }
 
+  /// Update the size of the texture.
```

Comment:
A suggestion for documentation readability: mention that the method accepts new texels along with the new size. This is obvious on inspecting the argument list, but the wording might mislead someone scanning for available options.

If I were writing this, I might put it as:
"Upload pixels to the whole texture, with a different size than previously specified."
(reusing the wording from `upload()`)

Reply:
I’m currently changing the API of textures so yes, it’s planned. :)